### PR TITLE
timeseries: remove Polymer based line chart

### PR DESCRIPTION
**Pull request by stephanwlee**
This change removes vz-line-chart based TimeSeries scalar chart. With
2.5.0, we released the Gpu line chart with an option to downgrade
temporarily to use the older behavior.

After the release, we now want to remove our tech debts by removing the
old code paths and update the tests accordingly.

After this change, we still have to:
1. remove tb-line-chart.
2. remove the feature flag.

### CODE REVIEW

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts [hunk 1]
```diff
@@ -608,192 +576,13 @@ describe('scalar card', () => {
     }
   });
 
-  it('uses proper default tooltip columns', fakeAsync(() => {
```
Comment (stephanwlee) [inline tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts:611]: Already covered by the specs previously under "gpu line chart".

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts [hunk 2]
```diff
@@ -608,192 +576,13 @@ describe('scalar card', () => {
...
-        results: ['10', '11', '12'],
-      },
-      {
-        title: 'Relative',
-        results: ['0s', '5s', '0s'],
-      },
-    ]);
-  }));
-
-  it('respects settings from the store', fakeAsync(() => {
```
Comment (stephanwlee) [inline tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts:663]: Sort and smoothing are tested covered below (prev gpu line chart section).

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts [hunk 3]
```diff
@@ -608,192 +576,13 @@ describe('scalar card', () => {
...
-
-    visibleRunIds = lineChart.seriesDataList
-      .filter((x) => x.visible)
-      .map((x) => x.seriesId);
-    expect(lineChart.seriesDataList.length).toBe(3);
-    expect(visibleRunIds).toEqual(['run3']);
-  }));
-
-  describe('controls', () => {
-    it('resets domain when user clicks on reset button', fakeAsync(() => {
```
Comment (stephanwlee) [inline tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts:792]: simple move under the `fit to domain` spec describes.

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts [hunk 4]
```diff
@@ -908,39 +648,8 @@ describe('scalar card', () => {
     }));
   });
 
-  describe('resize', () => {
```
Comment (stephanwlee) [inline tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts:911]: irreverent now

**Comment by stephanwlee** [comment]
Reviewer: you may want to enable/disable the ignore white space changes mode to filter out noises.

**Review by psybuzz — APPROVED** [review]
Thanks for this!  `getIsGpuChartEnabled` becomes unused in this PR, so presumably a followup will also remove it?

**Comment by stephanwlee** [comment]
Yup! The commit message says,

> After this change, we still have to:
> remove tb-line-chart.
> remove the feature flag.

remove the feature flag kinda meant removing the selector too.

**Pull request merged**